src/vl-upload.js

```diff
@@ -27,7 +27,7 @@ Promise.all([
  * @property {boolean} data-vl-full-body-drop - Attribuut om te activeren of deactiveren dat het de dropzone over het heel scherm is.
  * @property {string} data-vl-input-name - Attribuut om de key te definiëren waarmee het bestand wordt opgeladen.
  * @property {number} data-vl-max-files - Attribuut om het maximaal aantal bestanden dat opgeladen mag worden, aan te duiden.
- * @property {number} data-vl-max-size - Attribuut om de maximum grootte van een bestand dat opgeladen kan worden (20000000 = 2MB), aan te duiden.
+ * @property {number} data-vl-max-size - Attribuut om de maximum grootte van een bestand dat opgeladen kan worden (2097152 = 2MB), aan te duiden.
```

Comment:
Dit is dan niet meer backwards compatible. Maar het was kapot, dus dat maakt misschien niks uit, @wautergu...

Reply:
maar dus de verwachte eenheid is bytes nu, voor mij mag dat er nog wel expliciet bij

Reply:
Hangt ervan af hoe je het bekijkt. Hier gaat het enkel om de documentatie die verkeerd is: 20000000 komt overeen met (ongeveer) 20MB.
In code stond er als default 2000000 (bytes). Ik heb dat gewijzigd naar een echte 2MB. Normaal wordt er niets strenger gemaakt... enkel iets minder streng. Een gebruiker zal nu default een iets groter bestand kunnen opladen. 97152 bytes extra om precies te zijn.